example/spec/parallel/testAPI/granuleSpec.js

```diff
@@ -20,6 +21,7 @@ describe('The Granules API', () => {
   let config;
   let collection;
   let collectionId;
+  let discoveredGranule;
   let granuleId;
   let prefix;
   let granuleFile;
```

Comment:
Do you plan to get rid of buildRandomizedGranule method?

Reply:
I did and then forgot. thanks.

Reply:
0bd7337